src/packages/end-to-end/package.json

```diff
@@ -16,11 +16,12 @@
 		"dev-mysql": "pnpm import-database-mysql && pnpm start-server",
 		"unpack": "tsx ./scripts/unpack.ts",
 		"test-auth": "jest --runInBand --forceExit -- api/auth",
+		"test-aws-cognito": "jest --runInBand --forceExit ---- api/aws-cognito",
+		"test-metadata": "jest --runInBand --forceExit ---- api/metadata",
 		"test-mysql": "export DATABASE=mysql && jest --runInBand -- api/mysql",
 		"test-postgres": "export DATABASE=postgres && jest --runInBand -- api/postgres",
 		"test-sqlite": "dotenv -e .env.sqlite -- jest --runInBand --forceExit ---- api/sqlite",
 		"test-scalars": "jest --runInBand --forceExit ---- api/scalars",
-		"test-aws-cognito": "jest --runInBand --forceExit ---- api/aws-cognito",
```

Comment:
Should we add a new `test-api` that just does:
```
"test-api": "jest --runInBand --forceExit ---- api",
```
and use that in CI, just so that if someone adds a new one but doesn't hook it all the way up like this it doesn't accidentally not run?

Reply:
Not a bad shout but many of the API tests rely on an end to end app. So adding the script would not work out the box.

Maybe we can add a new ticket to tidy up the pattern for test runs?

Reply:
Yup, sounds good!